src/client/src/orangehrmRecruitmentPlugin/components/EditInterviewAttachment.vue

```diff
@@ -143,7 +143,7 @@ export default {
       .get(this.data.id)
       .then(response => {
         const {data} = response.data;
-        this.currentFile = data.attachment.fileName;
+        this.currentFile = data.filename;
```

Comment:
Have you checked if this is impacting other screens?  this component is used in multiple other places.